src/components/sandboxed-html.jsx

```diff
@@ -31,7 +31,7 @@ export default class SandboxedHtml extends React.PureComponent {
 
   render () {
     return <iframe
-      sandbox="allow-forms allow-scripts"
```

Comment:
I think it might be good to add a test in `src/components/__tests__/sandboxed-html.jsx` to make sure the `sandbox` attribute has all of the right values in case someone tries to remove it in the future.

Reply:
Hmmmm, I *think* I’d actually like to be able to remove the `allow-popups` attribute in the future if we can (obviously it’s required in this implementation, but a more advanced future version of this could use `postMessage` to signal to the app that a new window should be opened, and let the app choose what to do—that’d let us do fancy things like open the diff view for the link target instead of popping a new window). It’d be nice to make sure pages we show in the diff view can’t randomly create popups at will.

At the very least, it would be helpful to add a comment explaining that `allow-popups` is required for the `addTargetBlank` transform.